src/pybind/mgr/orchestrator.py

```diff
@@ -197,6 +197,7 @@ def add_host(self, host):
         Add a host to the orchestrator inventory.
 
         :param host: hostname
+        :param role: Role to performn (osd, mon, mgr)
```

Comment:
add_host doesn't have a `role` parameter.

Reply:
Fixed (also it was in rm_host).